man/org.freedesktop.import1.xml

```diff
@@ -0,0 +1,279 @@
+      the result of the operation, which is one of <literal>done</literal> (on success),
+      <literal>canceled</literal> or <literal>failed</literal>.</para>
+    </refsect2>
+  </refsect1>
+
+  <refsect1>
+    <title>The Transfer Object</title>
+
+    <programlisting>
+$ gdbus introspect --system \
+        --dest org.freedesktop.import1 \
+        --object-path /org/freedesktop/import1/transfer/_1
+
+node /org/freedesktop/import1/transfer/_1 {
+  interface org.freedesktop.import1.Transfer {
+    methods:
+      Cancel();
+    signals:
+      LogMessage(u priority,
+                 s line);
+    properties:
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
+      readonly u Id = ...;
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
+      readonly s Local = '...';
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
+      readonly s Remote = '...';
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
+      readonly s Type = '...';
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
+      readonly s Verify = '...';
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("false")
+      readonly d Progress = ...;
+  };
+  interface org.freedesktop.DBus.Peer { ... };
+  interface org.freedesktop.DBus.Introspectable { ... };
+  interface org.freedesktop.DBus.Properties { ... };
+};
+    </programlisting>
+
+    <!--signal LogMessage is not documented!-->
+
+    <refsect2>
+      <title>Methods</title>
+
+      <para>The <function>Cancel()</function> method may be used to cancel the transfer. It takes no
+      parameters. This call is pretty much equivalent to the <function>CancelTransfer()</function> call on
+      the <structname>Manager</structname> interface (see above), but is exposed on the
+      <structname>Transfer</structname> object itself instead of taking a transfer ID.</para>
+    </refsect2>
+
+    <refsect2>
+      <title>Properties</title>
+
+      <para>The <varname>Id</varname> property exposes the numeric transfer ID of the transfer object.</para>
+
+      <para>The <varname>Local</varname>, <varname>Remote</varname>, <varname>Type</varname> properties
+      expose the local container name of this transfer, the remote source (in case of download: the URL, in
+      case of import/export a string describing the file descriptor passed in), and the type of operation
+      (see the Manager's <function>ListTransfer()</function> call above for an explanation of the possible
+      values).</para>
+
+      <para>The <varname>Verify</varname> property exposes the selected verification setting, and is only
+      defined for download operations (see above).</para>
+
+      <para>The <varname>Progress</varname> property exposes the current progress of the transfer, as a value
+      between 0.0 and 1.0. To show a progress bar on screen it recommend to query this value in regular
+      intervals, for example every 500 ms or so.</para>
```

Comment:
```suggestion
      intervals.</para>
```

Reply:
Giving an example interval doesn't really seem to provide much added value here.

Reply:
It gives a hint what interval is expected. 500 ms tells the reader that it doesn't have to be once every minute...

Reply:
I don't know, it feels pretty weird to recommend a specific polling interval for updating a progress bar. It seems oddly specific. Then again, it doesn't hurt to have it there either so I'm fine with leaving this as it is.